pkgs/top-level/all-packages.nix

```diff
@@ -12055,6 +12055,7 @@ with pkgs;
 
   inherit (callPackages ../development/compilers/nim { })
     nim-unwrapped nimble-unwrapped nim;
+  nimPackages = recurseIntoAttrs nim.pkgs;
```

Comment:
maybe nimPackages should be a scope, which would allow to override nim, nimble or nim_builder.

Reply:
ah sorry it is one.